### DISCUSSION TITLE
conversation width limiting seems unnecessary

### DISCUSSION THREAD

**Discussion by modernNeo** [wireapp/wire-webapp#11151] [Ideas]
Why is there a max-width set for the `.messages` class?

https://github.com/wireapp/wire-webapp/blob/c6dc0d7fc558efd66caf8ae6584d8261e8d4ef7a/src/style/content/conversation/message-list.less#L27

When I load https://app.wire.com/?hl=en, the conversation does not take advantage of the full width of my monitor.

Same issue here: https://github.com/wireapp/wire-webapp/blob/32a74bfa402e496c0f84549a473608bb1c7e84ed/src/style/content/conversation/input-bar.less#L46

**Comment by rkapur-gds** [comment]
I agree, It's very annoying to have to scroll horizontally for large code blocks

**Reply by modernNeo** [reply]
for the time being, I have found a hack-solution:

1. download the [TamperMonkey](https://www.tampermonkey.net/) extention to your web browser
2. set the following rules
```
(function() {
    'use strict';

    // Your code here...
    document.getElementsByClassName('messages')[0].style.maxWidth = '100%'
    document.getElementsByClassName('conversation-input-bar-text')[0].style.maxWidth = '100%'
    document.getElementsByClassName('shadow-input')[0].style.maxWidth = '100%'
})();
```
3. Reload the wire page

**Reply by rkapur-gds** [reply]
I use the desktop/electron app where I modify the JS/CSS of the electron app by unpacking the `.asar` file, but it would be better if this was just fixed in the real app.

Thank you though!
